pages/08.scr/default.en.md

```diff
@@ -0,0 +1,42 @@
+---
+title: SysEleven Container Registry
+page-toc:
+    active: false
+body_classes: 'title-center title-h1h2'
+---
+
+[SysEleven Container Registry (SCR)](https://scr.syseleven.de) is the managed registry for containers and Helm charts from [SysEleven](https://www.syseleven.de) - a Berlin based managed hosting company.
+
+[SCR](https://scr.syseleven.de) is based on [Cloud Native Comuputing Foundation](cncf.io) incubated, open source project [Harbor](https://goharbor.io/)
+
+
+## Architectural Overview
+
+The following document explains in detail the architechture of [Harbor](https://goharbor.io/)
+
+![Harbor Architecture Overview](https://github.com/goharbor/harbor/wiki/Architecture-Overview-of-Harbor)
+
+## User Guide
```

Comment:
The docs sometime mention <harbor_url>. We should document what this is in our case.